frontend/packages/dev-console/src/components/pipelineruns/PipelineRunRow.tsx

```diff
@@ -47,9 +47,6 @@ const PipelineRunRow: React.FC<PipelineRunRowProps> = ({ obj, index, key, style
       </TableData>
       <TableData className={tableColumnClasses[4]}>-</TableData>
```

Comment:
Wow, there is another column here doing nothing.

Reply:
It's the duration field, which should come as a backend response. It is blank from the very beginning

Reply:
Can you log a bug for us to address it later. We'll investigate with UX to see if there was any intent here.